build/stf-run-ci/tasks/setup_base.yml

```diff
@@ -26,20 +26,42 @@
         targetNamespaces:
         - "{{ namespace }}"
 
-- name: Subscribe to AMQ Certificate Manager Operator
-  k8s:
-    definition:
-      apiVersion: operators.coreos.com/v1alpha1
-      kind: Subscription
-      metadata:
-        name: amq7-cert-manager-operator
-        namespace: openshift-operators
-      spec:
-        channel: 1.x
-        installPlanApproval: Automatic
-        name: amq7-cert-manager-operator
-        source: redhat-operators
-        sourceNamespace: openshift-marketplace
+- block:
```

Comment:
I was unable to get the Operator to install without pre-creating the namespace and OperatorGroup, which is why this is more verbose than the previous subscription, which used a pre-created namespace.